apps/io-wallet-user-func/src/app/main.ts

```diff
@@ -135,6 +139,7 @@ app.http("getCurrentWalletInstanceStatus", {
 app.http("setWalletInstanceStatus", {
   authLevel: "function",
   handler: SetWalletInstanceStatusFunction({
+    credentialRepository: pidIssuerClient,
```

Comment:
I wouldn't use `credentialRepository` as this is specific to the `pidIssuer`. There may also be other credential issuers other than the `pidIssuer`. I therefore recommend renaming `credentialRepository` to `pidIssuerRepository` or similar.

Reply:
`pidIssuer` or something similar would be wrong since those references are inside `infra`. I would say to leave it as it is and then change it later since it is correct for now